action.yml

```diff
@@ -35,8 +35,18 @@ inputs:
     required: false
     description: "Interval to poll Platform for application details. Default is once a minute."
     default: "60000"
+  enable_sarif:
+    required: false
+    description: "Enable creation of a SARIF file."
+    default: true
+  enable_dependencies:
+    required: false
+    description: "Enable posting of dependencies results to GitHub."
+    default: false
+  github_token:
```

Comment:
Since all the other inputs specify `required` explicitly, it should probably also be done here for consistency.